catalog/templates/catalog/course.html

```diff
@@ -120,6 +121,7 @@ <h1 class="d-flex align-items-center gap-2">
                         </div>
                         <div>
                             <div class="hover-underline">
+                                {% if document.hidden %}<strong>Only visible for you: this document is hidden</strong><br>{% endif %}
```

Comment:
The message should be in french to be consistent with the rest of the UI.

I'm also thinking that it might be best to keep the same number of lines than the other documents. Maybe we could show an icon next to the title ? This one seems appropriate https://icons.getbootstrap.com/icons/eye-slash-fill/ (with a title="Ce document a été caché, il est uniquement visible pour son propriétaire et aux modérateurs")

Reply:
the title is on the link of the document since the hover of it take all the <li> element

Reply:
![image](https://github.com/DocHub-ULB/DocHub/assets/43477020/d37260b7-54c5-452a-a312-b8b8b3de3103)